src/styles/navbar.css

```diff
@@ -0,0 +1,29 @@
+.navbar {
+  width: 100%;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: rgb(247, 247, 247);
+  justify-content: space-between;
+  border-bottom: solid 1px var(--grey-border);
+}
+
+#logo {
+  width: 18rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  letter-spacing: normal;
+}
+
+.navmenu {
+  list-style: none;
+}
+
+.navmenu a {
+  color: #121212;
+  margin-left: 40px;
+  text-decoration: none;
+}
```

Comment:
- [ ] Hi @Mesi21, nice job on your styling so far. Please can we improve on the styling of the navbar?. Styling should be EXACTLY  the same as the one provided on Zeplin. Color, font size, font weight, and layout should be EXACTly the same as the one in Zeplin. The background color of the navbar should be white, The user icon should be aligned with the main component as seen in the screenshots below.

Your Implementation | Figma Zeplin
:---:|:---:
![](https://user-images.githubusercontent.com/63087491/167233865-5aacdd6a-6d4a-4867-b431-68ed9f9dfe80.png) | ![](https://user-images.githubusercontent.com/63087491/167233881-85c69883-9b4e-4b5a-95bd-87e7bafb7742.png)